### PR TITLE
Create GitHub workflow

### PR DESCRIPTION
**Pull request by jleber**
#  :construction: Project CI/CD

This PR contains the inclusion of GitHub actions in order to have a CI/CD

# :repeat: Steps to test

Take a look at https://github.com/pipefy/pipefy_message/actions, and check the jobs running logs and the definition of each one. Also, you can fork this branch and open a new PR to trigger the workflow or do a new commit at this one.

# 🗂 Related cards

[[async] Create a pipeline and release flow for the gem project](https://app.pipefy.com/open-cards/490516417)

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+name: pull-request
+
+
```
Comment (viniciuscbm) [inline .github/workflows/pull-request.yml:3]: We could to remove one line here? :nail_care:

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Chekout repo
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
```
Comment (viniciuscbm) [inline .github/workflows/pull-request.yml:37]: We should use 2.6.6 here?

File: .github/workflows/pull-request.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        run: gem build pipefy_message.gemspec
+
+
+  test:
+    name: Unit Test
+    needs: [ build ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Chekout repo
```
Comment (viniciuscbm) [inline .github/workflows/pull-request.yml:61]: Chekout -> Checkout

File: .github/workflows/pull-request.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
```
Comment (viniciuscbm) [inline .github/workflows/pull-request.yml:67]: 2.6.6 here too?

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,89 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
```
Comment (viniciuscbm) [inline .github/workflows/release.yml:6]: I think the branch is main.

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+jobs:
+  build:
+    name: Build App
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Chekout repo
```
Comment (viniciuscbm) [inline .github/workflows/release.yml:14]: Chekout -> Checkout

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
```
Comment (viniciuscbm) [inline .github/workflows/release.yml:20]: 2.6.6 here too?

File: .github/workflows/release.yml [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+      - name: Build Gem
+        run: gem build pipefy_message.gemspec
+
+  test:
+    name: Unit Test
+    needs: [ build ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Chekout repo
```
Comment (viniciuscbm) [inline .github/workflows/release.yml:43]: Chekout -> Checkout

File: .github/workflows/release.yml [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
```
Comment (viniciuscbm) [inline .github/workflows/release.yml:49]: 2.6.6 here too?

File: .github/workflows/pull-request.yml [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+      - name: Run Rubocop
+        run: rubocop
+
+  build:
+    name: Build App
+    needs: [ lint ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Chekout repo
```
Comment (viniciuscbm) [inline .github/workflows/pull-request.yml:31]: Chekout -> Checkout

**Review by viniciuscbm — APPROVED** [review]

**Review by looslidias — APPROVED** [review]

**Pull request merged**